src/main/java/com/github/vkorobkov/jfixtures/processor/Processor.java

```diff
@@ -40,11 +41,19 @@ private void processFixture(Fixture fixture) {
     }
 
     private void handleFixtureInstructions(Fixture fixture) {
-        List<Instruction> fixtureInstructions = new ArrayList<>();
         val table = context.getConfig().table(fixture.name);
+        List<Instruction> fixtureInstructions = new ArrayList<>();
```

Comment:
Nope. I mean first you create `fixtureInstructions` at L45. Then at lines L49 and L53, L57 you add something to `fixtureInstructions`. Then you add all the `fixtureInstructions` into `context.getInstructions()`.

The question is, why can't we add elements directly into `context.getInstructions()` at L49 and L53, L57?

I think I recall why - because one table can start processing, add cleanup instruction, add some inserts and then it turns out that the table has a dependency to another table and another table will start processing. But the instructions for a table should go in a single block. That's why we need this temporary buffer.

Though it is a bad design - we need to think how to make it more explicit. Leave it as is for now, let's take another look on it after other minor issues get fixed.